requirements-dev.txt

```diff
@@ -1,17 +1,17 @@
-coveralls==2.1.2
-crc32c==2.1
-docker-py==1.10.6
-flake8==3.8.3
-lz4==3.1.0
-mock==4.0.2
-py==1.9.0
-pylint==2.6.0
-pytest==6.0.2
-pytest-cov==2.10.1
-pytest-mock==3.3.1
-pytest-pylint==0.17.0
-python-snappy==0.5.4
-Sphinx==3.2.1
-sphinx-rtd-theme==0.5.0
-tox==3.20.0
-xxhash==2.0.0
+coveralls
+crc32c
+docker-py
+flake8
+lz4
+mock
+py
+pylint
+pytest
+pytest-cov
+pytest-mock
+pytest-pylint
+python-snappy
+Sphinx
+sphinx-rtd-theme
+tox
+xxhash
```

Comment:
In my personal opinion, we don't necessarily need to pin versions until we get all of our workflows in order. We can revisit version pinning at a later point, many of these pinned versions are outdated.